Video/Video/Models/VideoController.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  VideoController.swift
+//  Video
+//
+//  Created by Wyatt Harrell on 5/6/20.
+//  Copyright © 2020 Wyatt Harrell. All rights reserved.
+//
+
+import Foundation
+
+let videoController = VideoController()
```

Comment:
This def works for a quick & dirty "singleton," but I usually try to avoid these sorts of global variables, as it can potentially lead to confusion down the line in a larger project (eg if a viewController has a property called `videoController`, it could get confusing whether you're referring to this global `videoController` or `self.videoController`). I'd prefer to put a static property on the class itself instead, even though it might involve a few more characters (eg `VideoController.shared`).